README.md

```diff
@@ -0,0 +1,115 @@
+# SoPra FS22 - Group 04 Frontend
+
+## Introduction
+Group matching in university courses can be very tedious. Whether it is finding great group members, having similar group sizes
+or simply having balanced skills across all groups.
+In order to circumvent that we decided to create a group matcher which does exactly that: match groups according to customizable
+traits and group sizes with the help of a matching quiz!
+
+The matching quiz consists of single and multiple choice questions which are editable and predefined by the course administrators -
+of which you can have as many as you need! The quiz will then be sent out via e-mail to a customizable list of students/participants
+who can fill it out and submit their answers!
+
+Depending on the settings you implemented (group size etc.) the groups will be matched accordingly once everybody has submitted
+their answers or the deadline has been met - of course all participants will be notified via e-mail with their group and team members!
+
+The goal of this application is to support both teachers and students in group related work and provide everyone with a quick, easy
+and fair way of finding group members and organizing the surrounding structure.
+
+## Technologies
+- React: CSS, Typescript, HTML, Formik
+- Chakra UI
+- GitHub Actions
+- Heroku
+- REST API
+
+## High-level Components
+Most important components:
+- [Dashboard](src/pages/admin/Dashboard.tsx)
```

Comment:
Maybe adding the Overview page as well since Dashboard mostly just provide the sidebar?

Reply:
Sure will do.